app/src/main/java/ai/elimu/content_provider/room/db/RoomDb.java

```diff
@@ -62,7 +62,8 @@ public static RoomDb getDatabase(final Context context) {
                                     MIGRATION_2_3,
                                     MIGRATION_3_4,
                                     MIGRATION_8_9,
-                                    MIGRATION_9_10
+                                    MIGRATION_9_10,
+                                    MIGRATION_11_12
```

Comment:
What will happen when a device is upgraded from version 10 to 11 and there is no migration script?